proto/buf.yaml

```diff
@@ -1,6 +1,8 @@
 version: v1
 deps:
   - buf.build/cosmos/gogo-proto
+    # Latest release was in Dec 2021, https://buf.build/cosmos/cosmos-proto/commits/main
+  - buf.build/cosmos/cosmos-proto:1935555c206d4afb9e94615dfd0fad31
```

Comment:
Can we add some documentation to the `README.md` of this project around the following:

1. The fact that we are copy/pasting protos into this repo, and that they should stay in sync with `dydxprotocol/v4` for now (until we find a better solution).
2. The steps to follow to update them, compile them, and the steps to update the `buf.yaml file.